tests/Browser/Alpine/EntangleDefer/Test.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Browser\Alpine\EntangleDefer;
+
+use Livewire\Livewire;
+use Tests\Browser\TestCase;
+use Tests\Browser\Alpine\EntangleDefer\Component;
+
+class Test extends TestCase
+{
+    public function test()
+    {
+        $this->browse(function ($browser) {
+            Livewire::visit($browser, Component::class)
+                /**
+                 * Can defer multiple entangle changes until next server request
+                 */
+                ->assertSeeIn('@output.alpine', 'false')
+                ->assertSeeIn('@output.livewire', 'false')
+                ->click('@toggle')
+                ->pause(150)
```

Comment:
i dont think you need to pause here

Reply:
Thanks! Agreed. I was looking at some of the other tests when I wrote it, and I think I had a livewire test open at the time and it had pauses in it. I will fix both up.